include/gf_vect_mul.h

```diff
@@ -43,6 +43,11 @@
 extern "C" {
 #endif
 
+// aarch64 only
+#if defined(__aarch64__)
+int gf_vect_mul_neon(int len, unsigned char *gftbl, void *src, void *dest);
```

Comment:
Same goes for this function as previous.  We don't want to add any new arch-specific exports that are planned for removal.

Reply:
> Same goes for this function as previous. We don't want to add any new arch-specific exports that are planned for removal.

remove arch-specific

Reply:
No not just the ifdef, we don't want any new exported versions.  Any current arch-specific versions are deprecated and are going to be removed.  The patch shouldn't need any changes to include/ directory.

Reply:
Done